starboard/contrib/cast/README.md

```diff
@@ -0,0 +1,53 @@
+## Cast Starboard API
+
+The Cast Starboard API is a shared library which contains the portion of
+Starboard required to run Cast.
+
+### Customizations
+
+As of Starboard 15, there are public methods required for Cast that are not
+already exposed by `libstarboard_platform_group.so`. As a result, the dedicated
+header `cast_starboard_api.h` is omitted from this release, though it may
+return in the future.
+
+Cast still requires additional behavior be implemented behind the existing
+Starboard APIs. Reference the `Cast TV Integration Guide` for details.
+
+### Reference Implementation
+
+The `cast_starboard_api/samples/` directory contains the reference target
+`cast_starboard_api` which can be built when
+`build_with_separate_cobalt_toolchain=true` is specified. To generate the
```

Comment:
I would also add the new variable. So something like
`build_with_separate_cobalt_toolchain=true` `cast_sample=true`

Reply:
Added `use_contrib_cast`